### DISCUSSION TITLE
thanhle.blog/webpack-babel-tui-no-lam-gi-o-day

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#50] [Ideas]
# thanhle.blog/webpack-babel-tui-no-lam-gi-o-day

Tìm hiểu về các công cụ được dùng trong một dự án frontend để biết cách tối ưu hơn

https://thanhle.blog/blog/webpack-babel-tui-no-lam-gi-o-day

**Comment by nguyenhoaikhang37** [comment]
bài viết hay quá, ủng hộ anh 1 tym <3

**Reply by thanhlmm** [reply]
haha thanks em!

**Comment by binhnguyen2501** [comment]
<3

**Comment by datistpham** [comment]
vấn đề mà mình thắc mắc bao nhiêu lâu nay đã được giải đáp, cảm ơn bạn nhiều nha

**Reply by thanhlmm** [reply]
thanks em. Nhớ subcribe email để nhận bài viết mới nhé :D
